colcon_cd/completer.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 Chen Bainian
+# Licensed under the Apache License, Version 2.0
+
+import argparse
+
+import argcomplete
+from colcon_argcomplete.argument_parser.argcomplete.completer.package_name \
+    import package_name_completer
+# from argcomplete.completers import SuppressCompleter
+
+
+def main():
+    """
+    Fake command with parser for colcon_cd completer.
+
+    Using --external-argcomplete-script to construct
+    a bash completer for the colcon_cd command.
+    """
+    parser = argparse.ArgumentParser()
+
+    # base_path argument is needed for package_name_completer
+    parser.add_argument(
```

Comment:
Use double quotes, as that seems to be the standard in this package.

Reply:
I think single quote is the standard here. As is in all of the test scripts.

Reply:
My bad, I don't know how I came to the double quote conclusion in the first place.